backend/dal/Services/Concrete/ParcelService.cs

```diff
@@ -226,8 +226,6 @@ public Parcel Update(Parcel parcel)
             // Do not allow switching agencies through this method.
             if (existingParcel.AgencyId != parcel.AgencyId) throw new NotAuthorizedException("Parcel cannot be transferred to the specified agency.");
 
-            this.Context.Parcels.ThrowIfNotUnique(parcel);
```

Comment:
Any reason to remove this?  The purpose is to handle the fact our unique identification relies on two fields.

Reply:
Yep, unique isn't possible when doing updates.

Reply:
We'll need to fix this to work then.  Otherwise they could update the PID or PIN to a value that already exists.